app/routes/quotes/components/AddQuote.tsx

```diff
@@ -30,7 +30,22 @@ const validate = createValidator({
 });
 
 const AddQuote = ({ addQuotes, actionGrant }: Props) => {
-  const onSubmit = withSubmissionErrorFinalForm(addQuotes);
+  const removeUnnecessaryDash = (source: string) => {
+    if (source === undefined) return undefined;
+
+    const dashIndex = source.indexOf('-');
+    if (source.slice(0, dashIndex).match(/^ *$/)) {
+      source = source.slice(dashIndex + 1);
```

Comment:
Could you also add a trim for spaces after the dash to prevent double spacing if someone inserts " -   Esso", to make the text consistent?

Reply:
I also thought about this! However, it seems like it does that already.
![image](https://user-images.githubusercontent.com/66320400/226112821-193bb0f3-a00b-4049-8325-d33428e1cb09.png)

Reply:
Ahh right, the browser does that for you when displaying it. Still wouldn't hurt to remove it to have cleaner data in the database, but that makes it less important

Reply:
Good point. I'll fix it right away